File: backend/src/main/java/com/wgzhao/addax/admin/repository/VwAddaxLogRepo.java
package com.wgzhao.addax.admin.repository;

import com.wgzhao.addax.admin.model.VwAddaxLog;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface VwAddaxLogRepo extends JpaRepository<VwAddaxLog, String> {
    List<VwAddaxLog> findTop15BySpnameIn(List<String> spNames);
}

File: src/main/kotlin/com/wgzhao/addax/admin/repository/VwAddaxLogRepo.kt
package com.wgzhao.addax.admin.repository

import com.wgzhao.addax.admin.model.VwAddaxLog
import org.springframework.data.jpa.repository.JpaRepository

interface VwAddaxLogRepo : JpaRepository<VwAddaxLog?, String?> {
    fun findTop15BySpnameIn(spNames: MutableList<String?>?): MutableList<VwAddaxLog?>?
}
